cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsInstanceConnector.java

```diff
@@ -168,6 +178,10 @@ private List<CloudVmInstanceStatus> setCloudVmInstanceStatuses(AuthenticatedCont
         }
     }
 
+    private void forceStopPendingInstances(AmazonEc2Client amazonEC2Client, Collection<String> pendingInstances) {
```

Comment:
Wanted to reconfirm: Is there no other action that can be performed on instances stuck in this state apart from `force stop`?

Reply:
Based on my check about it there are two options:-
1- force stop
2- terminate
As discussed we should not terminate and add a new instance in fast scaling so according to me force stop should be the best option.